platforms/software/dependency-management/src/main/java/org/gradle/api/internal/artifacts/transform/TransformStepNode.java

```diff
@@ -246,17 +244,8 @@ protected TransformStepBuildOperation createBuildOperation(NodeExecutionContext
                 return new TransformStepBuildOperation() {
                     @Override
                     protected TransformStepSubject transform() {
-                        TransformStepSubject initialSubject;
-                        try {
-                            initialSubject = TransformStepSubject.initial(artifact);
-                        } catch (ResolveException e) {
-                            throw e;
-                        } catch (RuntimeException e) {
-                            throw new DefaultLenientConfiguration.ArtifactResolveException("artifacts", transformStep.getDisplayName(), Collections.singleton(e));
-                        }
```

Comment:
`TransformStepSubject.initial(artifact)` just calls a constructor with no logic in it. It does not throw an exception.

In the past, the logic inside that block was more complicated and warranted a try/catch but now it will not throw.